src/faebryk/library/RP2040_ReferenceDesign.py

```diff
@@ -119,39 +108,16 @@ def __preinit__(self):
 
         # Flash
         self.flash.memory_size.constrain_subset(16 * P.Mbit)
-        self.flash.decoupled.decouple().capacitance.constrain_subset(
-            L.Range.from_center_rel(100 * P.nF, 0.05)
-        )
+        self.flash.decoupled.decouple().explicit(100 * P.nF)
 
         # Power rails
-        range_100nF = L.Range.from_center_rel(100 * P.nF, 0.05)
-        caps_100nF = []
-        multi_cap = F.MultiCapacitor(6)
-        self.rp2040.power_io.decoupled.decouple().specialize(
-            multi_cap
-        ).set_equal_capacitance_each(range_100nF)
-        caps_100nF.extend(multi_cap.capacitors)
-        self.rp2040.core_regulator.power_in.decoupled.decouple().capacitance.constrain_subset(
-            L.Range.from_center_rel(1 * P.uF, 0.05)
-        )
-        cap = self.rp2040.power_adc.decoupled.decouple()
-        cap.capacitance.constrain_subset(range_100nF)
-        caps_100nF.append(cap)
-        cap = self.rp2040.power_usb_phy.decoupled.decouple()
-        cap.capacitance.constrain_subset(range_100nF)
-        caps_100nF.append(cap)
-        power_3v3.decoupled.decouple().capacitance.constrain_subset(
-            L.Range.from_center_rel(10 * P.uF, 0.05)
-        )
-        multi_cap = F.MultiCapacitor(2)
-        self.rp2040.power_core.decoupled.decouple().specialize(
-            multi_cap
-        ).set_equal_capacitance_each(range_100nF)
-        caps_100nF.extend(multi_cap.capacitors)
-        self.rp2040.core_regulator.power_out.decoupled.decouple().capacitance.constrain_subset(
-            L.Range.from_center_rel(1 * P.uF, 0.05)
-        )
-
+        self.rp2040.power_io.decoupled.decouple(6).explicit(100 * P.nF)
```

Comment:
These MultiCap changes seem unrelated, and I think warrant more scrutiny, but we won't do it now because it's entangled with the performance updates. Let's get back to it later